examples/summarization/inference_compare.py

```diff
@@ -187,39 +188,42 @@ def reward_model(accelerator, model, inputs):
 
         with torch.no_grad():
             generated_sequences = generate_from_prompt(
-                inputs["input_ids_prompt"].to(accelerator.device),
-                inputs["attention_mask_prompt"].to(accelerator.device),
+                inputs["input_ids"].to(accelerator.device),
+                inputs["attention_mask"].to(accelerator.device),
                 model,
                 script_args,
             )
 
-            #TODO 
-            #create input ids that are right-padded with tokenizer.pad_token_id
-            #create corresponding attention mask
-            #create labels corresponding to the generated tokens (not including prompt) 
-            #should have -100 (label_pad_token) everywhere else
-            reward_model_input = {
-                "input_ids": None,
-                "attention_mask": generated_attention_mask,
-                "labels": None,
+            generated_texts = tokenizer.batch_decode(generated_sequences, skip_special_tokens=True)
```

Comment:
You might need to add a space between `TLDR:` and the continuation to be consistent with the supervised dataset. Can you look into this?

Reply:
There's already a space there in the `prompt`. I assumed that's the correct format. Right?

Reply:
The issue is that generation is much better if you don't start with a space (it seems like its an issue in Pythia). So you want to generate without a space `TLDR:` but then when you combine the prompt and completion, it's better to add the space so `TLDR:` + ` ` + `generation`

Reply:
You should double-check but just follow the format I used in https://huggingface.co/datasets/mnoukhov/openai_summarize_comparisons_relabel_pythia1b_tldrprompt?row=0 There might be no space at all